science/uhd/Portfile

```diff
@@ -234,35 +229,30 @@ default_variants +docs +examples +libusb +manual +test +manpages +gpsd
 
 # select Python to use; default to 2.7
 
-variant python27 conflicts python34 python35 python36 description {Build using Python 2.7} {}
-variant python34 conflicts python27 python35 python36 description {Build using Python 3.4} {}
-variant python35 conflicts python27 python34 python36 description {Build using Python 3.5} {}
-variant python36 conflicts python27 python34 python35 description {Build using Python 3.6} {}
+variant python27 conflicts python36 python37 description {Build using Python 2.7} {}
+variant python36 conflicts python27 python37 description {Build using Python 3.6} {}
+variant python37 conflicts python27 python36 description {Build using Python 3.7} {}
 
 if {![variant_isset python27] &&
-    ![variant_isset python34] &&
-    ![variant_isset python35] &&
-    ![variant_isset python36]} {
+    ![variant_isset python36] &&
+    ![variant_isset python37]} {
     default_variants +python27
```

Comment:
Sorry to add this request in late, but please switch the default to `+python37`.

Reply:
@michaelld: Are you OK with that change?

Reply:
Are you sure? because personally I prefer to maintain all *SDR* packages inline with GNU Radio python version, which is `python27`

Reply:
That makes sense to me. I think we want to keep them all at the same default.

Reply:
Right. Let's move all of that to a future PR/commit.